From 5d71dc5d5ba163bf5e46364fe0844b9f470e910d Mon Sep 17 00:00:00 2001
From: Matthias Klumpp <matthias@tenstral.net>
Date: Mon, 8 Sep 2014 17:38:59 +0200
Subject: [PATCH] Increase the default transaction limits

Those served us well in Debian, and the low values cause problems
sometimes.
This should be a sane default for all distributions, now that the yum
backend is gone.
---
 src/pk-transaction.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/pk-transaction.c b/src/pk-transaction.c
index e5f6fcd..f85a71c 100644
--- a/src/pk-transaction.c
+++ b/src/pk-transaction.c
@@ -70,10 +70,10 @@ static gboolean pk_transaction_is_supported_content_type (PkTransaction *transac
 #define PK_TRANSACTION_UID_INVALID		G_MAXUINT
 
 /* maximum number of items that can be resolved in one go */
-#define PK_TRANSACTION_MAX_ITEMS_TO_RESOLVE	1200
+#define PK_TRANSACTION_MAX_ITEMS_TO_RESOLVE	4800
 
 /* maximum number of packages that can be processed in one go */
-#define PK_TRANSACTION_MAX_PACKAGES_TO_PROCESS	5000
+#define PK_TRANSACTION_MAX_PACKAGES_TO_PROCESS	5200
 
 struct PkTransactionPrivate
 {
-- 
1.7.1

